NetCord/Gateway/GatewayClient.cs

```diff
@@ -17,72 +20,626 @@ public partial class GatewayClient : WebSocketClient, IEntity
     private readonly IGatewayCompression _compression;
     private readonly bool _disposeRest;
 
+    #region Events
+
+    /// <summary>
+    /// The ready event is dispatched when a client has completed the initial handshake with the Gateway (for new sessions).
+    /// The ready event, while generally small in practice, can be the largest and most complex event the Gateway will send, as it contains all the state required for a client to begin interacting with the rest of the platform.<br/>
```

Comment:
What does `while generally small in practice, can be the largest and most complex event the Gateway will send` mean? It's small but can be the largest? I think we shouldn't document it like that. I know that the discord documentation states that, but it's not true.

Reply:
Eh, good point, I'll change it